UppercaseRot13Transformer wrong class name used

jevgenijusr opened the pull request:
Regarding UppercaseRot13Transformer: please double check this, because I suppose UppercaseTransformer has to be used instead. We are autowiring UppercaseTransformer, also there is no description of UppercaseRot13Transformer at all. P.S. If I am wrong, then I apologize 100 times:), but in any case this means it can be difficult to understand part of the recipe

Commit:
UppercaseRot13Transformer wrong class name used

Regarding UppercaseRot13Transformer please double check this, because I suppose UppercaseTransformer has to be used instead. We are autowiring UppercaseTransformer, also there is no description of UppercaseRot13Transformer at all.

xabbuh commented on components/dependency_injection/autowiring.rst:
```diff
@@ -317,7 +317,7 @@ and a Twitter client using it:
                 autowire: true
 
             uppercase_rot13_transformer:
```
I think we should rename this to `uppercase_transformer`.

xabbuh commented:
@jevgenijusr This change looks good to me. Can you also update the following code blocks for the XML and PHP formats accordingly?

Commit:
Replaced uppercase_rot13_transformer with uppercase_transformer

Removed usage of uppercase_rot13_transformer, used uppercase_transformer instead. Changed in YAML, XML and PHP code blocks.

xabbuh commented:
👍 

Status: Reviewed

javiereguiluz commented:
@jevgenijusr this looks wrong. Thanks for fixing it!  However, shouldn't we use `Rot13Transformer` instead of `UppercaseTransformer` ? That's what's used in the first part of the article.

Commit:
Changed name UppercaseTransformer to UppercaseRot13Transformer

Changed name of UppercaseTransformer to UppercaseRot13Transformer because it makes more sense: in this part of the article we do either rot13 transform or uppercased rot13 transform, therefore we use Rot13Transformer and UppercaseRot13Transformer. Actually the only problem this article had before this PR is it had mispelled "class UppercaseTransformer" (line 238) , it should have been "class UppercaseRot13Transformer", I renamed everything to use UppercaseTransformer at first, but best name would be UppercaseRot13Transformer (smallest change too) :) Take a look, thanks.

jevgenijusr commented:
@javiereguiluz Thanks, I think we should use `UppercaseRot13Transformer` instead, small change in the end :)

WouterJ commented:
Hi @jevgenijusr! Thanks for fixing this bug. The article is already quite complex, let alone when there is a typo in it :)

I've merged your PR in the 2.8 version of the docs, as this feature was added in 2.8. I'll merge it in the newer versions from there. After merging, I decided to rename `UppercaseRot13Transformer` to `UppercaseTransformer`: The class is actually a decorator that simply uppercases the value of any transformer passed in the constructor. You can see this change (among some other changes) in 048b853

Thanks again!
